pyproject.toml

```
@@ -107,7 +107,7 @@ gpu = ["cupy"]
 cu12 = ["cupy-cuda12x"]
 cu11 = ["cupy-cuda11x"]
 # https://github.com/dask/dask/issues/11290
-dask = ["dask[array]>=2022.09.2,<2024.8.0"]
+dask = ["dask[array]>=2022.09.2,!=2024.8.0,!=2024.8.1,!=2024.8.2,!=2024.9.0,!=2024.9.1"]
```

OK, so we *can* make it more elegant!

```suggestion
dask = ["dask[array]>=2022.09.2,!=2024.8.*,!=2024.9.*"]
```

```pycon
>>> spec = packaging.specifiers.SpecifierSet(">=2022.09.2,!=2024.8.*,!=2024.9.*")
>>> spec.contains('2024.7.0')
True
>>> spec.contains('2024.8.0')
False
>>> spec.contains('2024.8.1')
False
>>> spec.contains('2024.9.0')
False
>>> spec.contains('2024.10.0')
True
```